### DISCUSSION TITLE
2022/02/14/Hexo-Butterfly-%E5%BB%BA%E7%AB%99%E6%8C%87%E5%8D%97%EF%BC%88%E4%BA%8C%EF%BC%89%E9%83%A8%E7%BD%B2%E8%87%B3%E4%B8%AA%E4%BA%BA%E7%AB%99%E7%82%B9%E6%88%96-GithubPages/

### DISCUSSION THREAD

**Discussion by giscus** [nickxudotme/nickxudotme.github.io#27] [General]
# 2022/02/14/Hexo-Butterfly-%E5%BB%BA%E7%AB%99%E6%8C%87%E5%8D%97%EF%BC%88%E4%BA%8C%EF%BC%89%E9%83%A8%E7%BD%B2%E8%87%B3%E4%B8%AA%E4%BA%BA%E7%AB%99%E7%82%B9%E6%88%96-GithubPages/

现在我们需要把静态文件上传到公网上，让所有人都能访问 打开_config.yml，拉到最底下，找到# Deployment那一块，现在我们需要配置这里  按照官方文档有很多种方法，一般情况下只用得着两种：SFTP和Git SFTP推送至个人服务器是付费方案，Git推送至GitHub是免费方案 法一：

https://nickxu.me/2022/02/14/Hexo-Butterfly-%E5%BB%BA%E7%AB%99%E6%8C%87%E5%8D%97%EF%BC%88%E4%BA%8C%EF%BC%89%E9%83%A8%E7%BD%B2%E8%87%B3%E4%B8%AA%E4%BA%BA%E7%AB%99%E7%82%B9%E6%88%96-GithubPages/

**Comment by YXZ252426** [comment]
2024.12.19  现在部署到github pages似乎会不需要手动安装SSL证书
